modules/brightMountainMediaBidAdapter.js

```diff
@@ -108,6 +111,10 @@ export const spec = {
       if (bid.schain) {
         placement.schain = bid.schain;
       }
+
+      if (bid.userId) {
+        placement.userIds = { eids: createEidsArray(bid.userId) };
```

Comment:
why not just use the `bid.userIdAsEids` value instead of calling this function?

Reply:
Sounds good. I have updated adapter as per your suggestion. Can you have a look now ? @smenzer